### DISCUSSION TITLE
<FATAL>: unknown option: --allow-other

### DISCUSSION THREAD

**Discussion by solracsf** [juicedata/juicefs#2579] [General]
On JuiceFS v1.0.0, mounting with `--allow-other`:

`juicefs[31112] <FATAL>: unknown option: --allow-other [main.go:215]`

```
/usr/local/bin/juicefs mount \
    --no-usage-report \
    --backup-meta 0 \
    --cache-size 2048 \
    --cache-dir /srv/cache \
    --log /srv/juicefs.log \
    --get-timeout 900 \
    --put-timeout 900 \
    --io-retries 90 \
    --writeback \
    --upload-delay 20s \
    --allow-other \
    "mysql://juicefs:juicefs@unix(/run/mysqld/mysqld.sock)/juicefs" \
    /srv/juicefs
```
With or without `user_allow_other` option set on `/etc/fuse.conf`
System is `Ubuntu 22.04.1` on `arm64` with Kernel `5.15.0-46-generic`

**Comment by davies** [comment]
This is a FUSE option, so it should be `-o allow-other`

**Reply by solracsf** [reply]
Thanks. But it is not clear at https://juicefs.com/docs/cloud/commands_reference/#synopsis-1 🤔

**Reply by solracsf** [reply]
Ok I was reading the wrong page...
Good one is here https://juicefs.com/docs/community/fuse_mount_options
